programmers-coding-test/minjae/Implement/구현-게임개발.js

```diff
@@ -0,0 +1,52 @@
+function solution(n) {
+    const mapField = [
+        [1, 1, 1, 1],
+        [1, 0, 0, 1],
+        [1, 1, 0, 1],
+        [1, 1, 1, 1],
+    ];
+    const dx = [-1, 0, 1, 0];
+    const dy = [0, 1, 0, -1];
+
+    let x = n[0];
+    let y = n[1];
+    let direction = n[2];
```

Comment:
함수의 첫 인자로 x, y, direction을 받아오면 밑에서 따로 할당해주지 않아도 된다.
```suggestion
function solution(x, y, direction) {
    const mapField = [
        [1, 1, 1, 1],
        [1, 0, 0, 1],
        [1, 1, 0, 1],
        [1, 1, 1, 1],
    ];
    const dx = [-1, 0, 1, 0];
    const dy = [0, 1, 0, -1];
```